src/wallet/api/wallet.cpp

```diff
@@ -453,10 +453,20 @@ bool WalletImpl::createWatchOnly(const std::string &path, const std::string &pas
 }
 
 bool WalletImpl::recoverFromKeys(const std::string &path,
-                                const std::string &language,
-                                const std::string &address_string,
-                                const std::string &viewkey_string,
-                                const std::string &spendkey_string)
+                                 const std::string &language,
+                                 const std::string &address_string,
+                                 const std::string &viewkey_string,
+                                 const std::string &spendkey_string)
+{
```

Comment:
I don't have an opinion on the patch yet, but please stop adding random whitespace changes.

Reply:
if i'm going to touch it, i may as well make it look better - and consistent.

Reply:
It doesn't look modified (beyond the whitespace). Am I missing the obvious ?

Reply:
there are two new methods in wallet & wallet_manager & wallet2_api.h:
- recoveryWallet with a new password patameter
- recoverWithKeys also with a password parameter

the old methods recoveryWallet & recoverFromKeys call these new methods with an empty password to maintain backwards compatibility.

Reply:
Still these above 4 lines don't seem to be necessary to be changed. They can be kept as they are.

Reply:
no, but it looks better aligned.

Reply:
I think we prioritize more on smaller changes than cleaner look.